src/pages/Dashboard/components/DashboardHeader.tsx

```diff
@@ -32,54 +61,151 @@ export const DashboardHeader = () => {
     [claimableRewardsFiatBalance, earnFiatBalance, portfolioTotalFiatBalance],
   )
   const borderColor = useColorModeValue('gray.100', 'gray.750')
+  const bgColor = useColorModeValue('white', 'blackAlpha.100')
+
+  useEffect(() => {
+    if (activeRef.current) {
+      activeRef.current.scrollIntoView({ behavior: 'smooth', block: 'center', inline: 'center' })
+    }
+  }, [location])
+
+  const NavItems: TabItem[] = useMemo(() => {
+    return [
+      {
+        label: 'common.overview',
+        path: '/dashboard',
+        color: 'blue',
+        exact: true,
+      },
+      {
+        label: 'navBar.wallet',
+        path: '/dashboard/accounts',
+        color: 'blue',
+        fiatAmount: portfolioTotalFiatBalance,
+      },
+      {
+        label: 'navBar.earn',
+        path: '/dashboard/earn',
+        color: 'purple',
+        fiatAmount: earnFiatBalance,
+      },
+      {
+        label: 'navBar.rewards',
+        path: '/dashboard/rewards',
+        color: 'green',
+        fiatAmount: claimableRewardsFiatBalance,
+      },
+      {
+        label: 'NFTs',
+        path: '/dashboard/nfts',
+        color: 'pink',
+        hide: !isNftsEnabled,
+      },
+      {
+        label: 'navBar.activity',
+        path: '/dashboard/activity',
+        color: 'blue',
+      },
+    ]
+  }, [claimableRewardsFiatBalance, earnFiatBalance, isNftsEnabled, portfolioTotalFiatBalance])
+
+  const renderNavItems = useMemo(() => {
+    return NavItems.filter(item => !item.hide).map(navItem => (
+      <DashboardTab
+        key={navItem.label}
+        label={navItem.label}
+        path={navItem.path}
+        ref={location.pathname === navItem.path ? activeRef : null}
+        color={navItem.color}
+        fiatAmount={navItem.fiatAmount}
+        exact={navItem.exact}
+      />
+    ))
+  }, [NavItems, location.pathname])
+
+  const walletImage = useMemo(() => {
+    if (!walletId) return ''
+    return makeBlockiesUrl(`${walletId}ifyoudriveatruckdriveitlikeyouhaveafarm`)
```

Comment:
1) What

Reply:
lol -- the string needs to be at least 15 characters. So to make sure its always 15 we have a fun little snippet to throw in the generation.